workflow/wdl/workflow.wdl

```diff
@@ -0,0 +1,64 @@
+version 1.0
+
+# import "tasks/buildfile.wdl" as buildfile
+import "tasks/nextstrain.wdl" as nextstrain  # <= modular method
+# import "tasks/ncov_ingest.wdl" as ncov_ingest
+
+workflow Nextstrain_WRKFLW {
+  input {
+    # Option 1: Pass in a sequence and metadata files, create a build_yaml
+    File? sequence_fasta
+    File? metadata_tsv
+    String? build_name
```

Comment:
This might be a question for Theiagen office hours, but I wonder if we should eventually split these two options into separate workflows? Then we could more clearly define the sequence and metadata inputs as required for Option 1 and the builds config as required for option 2.

Reply:
Ah, I'm hoping this will also support when a user provides all three. :)

Reply:
I tried providing all three (sequences, metadata, with a customized config.yaml referencing the first two), seems to be running great ([view run](https://job-manager.dsde-prod.broadinstitute.org/jobs/6a4917f3-fa2a-4b36-8924-f5a138faddd7)). I checked logs and it could find and process all files.